pkg/cloudprovider/aws/launchtemplate.go

```diff
@@ -258,10 +256,34 @@ exec > >(tee /var/log/user-data.log|logger -t user-data -s 2>/dev/console) 2>&1
 	if len(constraints.KubeletConfiguration.ClusterDNS) > 0 {
 		userData.WriteString(fmt.Sprintf(` \
     --dns-cluster-ip '%s'`, constraints.KubeletConfiguration.ClusterDNS[0]))
+	}
+	if constraints.KubeletConfiguration.MaxPods < 0 {
+		userData.WriteString(` \
+    --use-max-pods=false`)
 	}
 	return base64.StdEncoding.EncodeToString(userData.Bytes()), nil
 }
 
+func (p *LaunchTemplateProvider) getKubeletExtraArgs(constraints *v1alpha1.Constraints, additionalLabels map[string]string) string {
+	var kubeletArgs bytes.Buffer
```

Comment:
Thoughts on just concatting to a string? Not sure I understand the point of using a byte buffer instead.

Reply:
No idea either. It was just the pattern being used before and continued it. 😆